packages/@okta/vuepress-site/docs/guides/configure-access-policy/main/index.md

```diff
@@ -50,9 +49,9 @@ Use the following steps to create a policy that limits which scopes that a desig
 
 ### Create the policy container
 
-1. In the Admin Console, go to **Security** > **API**.
+1. Go to **Security** > **API** in the Admin Console.
```

Comment:
Not really a big deal but our admin docs convention is the version you changed. Do we want this to match?

In the Admin Console, go to...